editor/effects.js

```diff
@@ -113,6 +115,10 @@ export default {
 			) );
 		}
 
+		// Update dirty metaboxes.
+		const metaboxes = getDirtyMetaboxes( getState() );
+		metaboxes.forEach( metabox => dispatch( requestMetaboxUpdate( metabox ) ) );
```

Comment:
Do we have to dispatch these individually, or can we just create a single action which will mark all as needing update?

Reply:
Not sure what you mean exactly.

Reply:
Something like:

```diff
		const metaboxes = getDirtyMetaboxes( getState() );
-		metaboxes.forEach( metabox => dispatch( requestMetaboxUpdate( metabox ) ) );
+		dispatch( requestMetaboxUpdates( metaboxes );
```

```diff
 		case 'REQUEST_META_BOX_UPDATE':
 			return { ...state, [ action.location ]: { ...state[ action.location ], isUpdating: true, isDirty: false } };
+
+		case 'REQUEST_META_BOX_UPDATES':
+			return action.locations.reduce( ( newState, location ) => {
+				newState[ location ] = {
+					...state[ location ],
+					...isUpdating: true,
+					...isDirty: false,
+				};
+				return newState;
+			}, { ...state } );
```

Reply:
Now I understand, thank you. Yes, we can do something like this, makes sense.